files-community/Files
# Feature: Display custom icon when Files is the registered handler for archives

**Resolved / Related Issues**
   Closes #12713...

**Validation**
How did you test these changes?
   1. Set Files as the registered handler for a zip file
   2. Confirm the custom icon is displayed instead of the usual app icon

**Screenshots (optional)**
Add screenshots here.
